src/selectors/prometheus/storage.js

```diff
@@ -0,0 +1,42 @@
+import { flatMap, max } from 'lodash';
+
+import { parseNumber, formatBytes } from '../../utils';
+
+export const getTopConsumerVectorStats = result => {
+  let maxVal = 0;
+  let flattenedResult = [];
+
+  // returns all the namespace.values in an array
+  flattenedResult = flatMap(result, namespace => namespace.values);
```

Comment:
you can name the variables accordingly and we might not need the comments
e.g.

```js
namespaceValues =
allValues = // allBytes ?

```

Reply:
ack